webapp-django/crashstats/base/views.py

```diff
@@ -27,5 +26,5 @@ def handler404(request):
             'path': request.path,
             'query_string': request.META.get('QUERY_STRING'),
         }, status=404)
-    context = {'product': settings.DEFAULT_PRODUCT}
+    context = {}
     return render(request, '404.html', context, status=404)
```

Comment:
Er, let me clarify. The 500 stuff always takes an empty context. The 404 stuff takes a context. I have no idea if the socorro 404 stuff cares about having the product in the context or not. If not, then it's moot, I suppose.